### PR TITLE
Fix hog scenarios to use the correct configuration file

### PR DESCRIPTION
**Pull request by chaitanyaenr**

### CODE REVIEW

File: cpu-hog/prow_run.sh [hunk 1]
```diff
@@ -15,19 +15,14 @@ SCENARIO_FOLDER="$krkn_loc/scenarios/arcaflow/cpu-hog"
 echo "Printing cluster version"
 oc version
 
-# Copy config to kraken
-cp node-cpu-hog/input.yaml.template $SCENARIO_FOLDER/input.yaml.template
-
 source node-cpu-hog/env.sh
-source env.sh
```
Comment (paigerube14) [inline cpu-hog/prow_run.sh:22]: Think we need this file

File: cpu-hog/prow_run.sh [hunk 2]
```diff
@@ -15,19 +15,14 @@ SCENARIO_FOLDER="$krkn_loc/scenarios/arcaflow/cpu-hog"
...
 source common_run.sh
 
 setup_arcaflow_env "$SCENARIO_FOLDER"
 checks
 
 # Substitute config with environment vars defined
-#envsubst < node-cpu-hog/input.yaml.template> node-cpu-hog/cpu_hog_scenario.yaml
```
Comment (paigerube14) [inline cpu-hog/prow_run.sh:29]: Why don't we need this scenario file anymore?

Comment (paigerube14) [reply cpu-hog/prow_run.sh:29]: Is it already set in the set arca scenario above?

File: cpu-hog/prow_run.sh (hunk 1 above)
Comment (chaitanyaenr) [reply cpu-hog/prow_run.sh:22]: We are already sourcing the env.sh file in https://github.com/redhat-chaos/prow-scripts/blob/main/cpu-hog/prow_run.sh#L8

File: cpu-hog/prow_run.sh (hunk 2 above)
Comment (chaitanyaenr) [reply cpu-hog/prow_run.sh:29]: oh my bad, setup_arcaflow_env does the envsubst.

**Review by paigerube14 — APPROVED** [review]
/lgtm

**Pull request merged**